ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2581,6 +2582,36 @@ def get_healthcheck_version(host):
     return healthcheck_version
 
 
+def wait_for_ipa_to_start(host, timeout=60):
+    """Wait up to timeout seconds for ipa to start on a given host.
+
+    If DS is restarted, and SSSD must be online, please consider using
+    wait_for_sssd_domain_status_online(host) in the test after calling
+    this method.
+    """
+    interval = 1
+    end_time = datetime.now() + timedelta(seconds=timeout)
+    for _i in range(0, timeout, interval):
+        if datetime.now() > end_time:
+            raise RuntimeError("Request timed out")
+        time.sleep(interval)
+        result = host.run_command(
+            [paths.IPACTL, "status"]
+        )
+        if result.returncode == 0:
+            break
+
+
+def dns_update_system_records(host):
+    """Runs "ipa dns-update-system-records" on "host".
+    """
+    kinit_admin(host)
+    result = host.run_command(
+        ["ipa", "dns-update-system-records"]
+    )
+    return result
```

Comment:
The result is never used in the calling methods: you can either add raiseonerr=False to the run_command or not return result.

Reply:
raiseonerr=False